cardano-tracer/src/Cardano/Tracer/Handlers/RTView/Utils.hs

```diff
@@ -1,23 +1,27 @@
 {-# LANGUAGE NamedFieldPuns #-}
 
 module Cardano.Tracer.Handlers.RTView.Utils
-  ( forAcceptedMetrics_
-  , forAcceptedMetricsUI_
-  , forConnected
-  , forConnected_
-  , forConnectedUI
-  , forConnectedUI_
+  ( module Cardano.Tracer.Handlers.RTView.Utils
```

Comment:
You like this crazy stuff

Reply:
Yes, I want to be explicit about: this module exports *every* definition and type it contains :)